.github/workflows/baselines.yml

```
@@ -54,7 +54,7 @@ jobs:
             # Pull request event triggered for any commit to a pull request
             change_references="main..HEAD"
           fi
-          dirs=$(git diff --dirstat=files,0 ${change_references} . | awk '{print $2}' | grep -E '^baselines/[^/]*/$' | grep -v -e '^baselines/dev' -e '^baselines/baseline_template' -e '^baselines/flwr_baselines' | sed 's/^baselines\///')
+          dirs=$(git diff --dirstat=files,0 ${change_references} . | awk '{print $2}' | grep -E '^baselines/[^/]*/$' | grep -v -e '^baselines/dev' -e '^baselines/baseline_template' -e '^baselines/flwr_baselines' -e '^baselines/doc' | sed 's/^baselines\///')
```

```suggestion
          dirs=$(git diff --dirstat=files,0 ${change_references} . | awk '{print $2}' | grep -E '^baselines/[^/]*/$' | \
          grep -v \
            -e '^baselines/dev' \
            -e '^baselines/baseline_template' \
            -e '^baselines/flwr_baselines' \
            -e '^baselines/doc' \
          | sed 's/^baselines\///')
```

@adam-narozniak would this also work to make it more readable?

---

@tanertopal yes, I check it also on my repo for tests too